xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-handlers/xwiki-platform-extension-handler-xar/src/test/java/org/xwiki/extension/xar/internal/handler/WikiCopiedEventListenerTest.java

```diff
@@ -86,6 +87,15 @@ public void setUp() throws Exception
 
         this.localExtensionDependency1 = this.localExtensionRepository.storeExtension(extensionDependency);
         this.localExtension1 = this.localExtensionRepository.storeExtension(extension);
+
+        this.repositoryUtil.getComponentManager().unregisterComponent(EventListener.class,
```

Comment:
I feel that registering a mock of ModelBridge (unless one already exist) in MockitoOldcore like we do for other components would be better and easier for new tests than copy/pasting this everywhere.

Reply:
I agree that it'll be more elegant, I'm working on that. However it won't be enough: here for example, we're not using MockitoOldCore. Now I noticed that I have less mock component to register, than listener to unregister, so I'll go toward this solution

Reply:
> we're not using MockitoOldCore

Sure but in many of the affected test we do and it's even possible some of those not using MockitoOldCore should.

Reply:
> t's even possible some of those not using MockitoOldCore should (or should not use `@AllComponent`).

yep, but I don't necessarily plan to refactor them ;)